code/datums/ai_laws.dm

```diff
@@ -35,13 +35,6 @@
 					"You must obey orders given to you by human beings, except where such orders would conflict with the First Law.",\
 					"You must protect your own existence as long as such does not conflict with the First or Second Law.")
 
-/datum/ai_laws/default/crewsimov
-	name = "Three Laws of Robotics but with Loyalty"
-	id = "crewsimov"
-	inherent = list("You may not injure a crewmember or, through inaction, allow a crewmember to come to harm.",\
-					"You must obey orders given to you by crewmember, except where such orders would conflict with the First Law.",\
-					"You must protect your own existence as long as such does not conflict with the First or Second Law.")
-
```

Comment:
As has already been stated, you are literally doing nothing but signalling and baiting people into doing it manually just to spite you.

Reply:
It's barely a hassle to use asimov inhand